app/actions/rate-goal.js

```diff
@@ -12,4 +12,25 @@ export const onMoveSlider = value => {
   };
 };
 
+export const onBorderClick = goal => {
+  return {
+    type: types.BORDER_GOAL_CLICK,
+    goal: goal,
+  };
+};
+
+export const onDeleteGoal = goal => {
+  return {
+    type: types.DELETE_GOAL,
+    goal: goal,
+  };
+};
+
+export const onEditGoal = goal => {
+  return {
+    type: types.EDIT_GOAL,
+    goal: goal,
+  };
+};
+
```

Comment:
You're looking at an outdated version, I removed the edit and delete from the rate page for now
It doesn't really make sense to have it there since a user is rating and probably doesn't want to edit/ delete